src/cache.rs

```diff
@@ -918,6 +918,11 @@ impl Cache {
         Some(file)
     }
 
+    pub fn get_imports(&self, file: &FileId) -> Option<Vec<FileId>> {
+        let s = self.imports.get(file)?;
```

Comment:
Nitpick: what does `s` mean? A longer name might more explicit :slightly_smiling_face:

Reply:
Forgot about that while cleaning up. Thanks 🙂